dashboards/istio-envoy/README.md

```diff
@@ -1,10 +1,10 @@
-### Dashboards for Envoy Proxy
+### Dashboards for Istio Envoy Proxy
```

Comment:
Can you also name the parent folder just `istio` - it's possible there will be other istio dashboards for components other than envoy in the future.

Reply:
done.